pkgs/by-name/tr/tracy/package.nix

```
@@ -60,7 +66,7 @@ stdenv.mkDerivation rec {
     make -j $NIX_BUILD_CORES -C csvexport/build/unix release
     make -j $NIX_BUILD_CORES -C import-chrome/build/unix release
     make -j $NIX_BUILD_CORES -C library/unix release
-    make -j $NIX_BUILD_CORES -C profiler/build/unix release LEGACY=1
+    make -j $NIX_BUILD_CORES -C profiler/build/unix release ${if withWayland then "" else "LEGACY=1"}
```

This should only affect Linux builds:

```suggestion
    make -j $NIX_BUILD_CORES -C profiler/build/unix release \
      ${lib.optionalString (stdenv.isLinux && !withWayland) "LEGACY=1"}
```